packages/custom_lint/test/create_project.dart

```diff
@@ -37,6 +38,7 @@ class TestLintRule {
   final String onVariable;
   final String ruleMembers;
   final List<TestLintFix> fixes;
+  final String errorSeverity;
```

Comment:
Don't use a String for this. Either make an enum or use the official ErrorSeverity object.

Reply:
fixed commit 8fffd8d